Branch-SDK/src/io/branch/referral/BranchPreinstall.java

```diff
@@ -0,0 +1,105 @@
+package io.branch.referral;
```

Comment:
Branch classes are formatted with 4 spaces.   Not a huge issue, but this would be an outlier.

Reply:
Done